<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remapper</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="val">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-style:solid;
border-width: 1px;
border-color: grey;
background-color: #edeae8;
margin-top:2px;
border-radius : 5px;
}

QLineEdit:hover{
	background-color : #e0e0e0;
}


QLineEdit:focus{
	background-color:#e0e0e0;
	border-width : 3px;
	border-color:#f2ce57;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Value Input</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="low1">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
border-style:solid;
border-width: 1px;
border-color: grey;
background-color: #edeae8;
margin-top:2px;
border-radius : 5px;
}

QLineEdit:hover{
	background-color : #e0e0e0;
}


QLineEdit:focus{
	background-color:#e0e0e0;
	border-width : 3px;
	border-color:#f2ce57;
}</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Low Value 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="high1">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
border-style:solid;
border-width: 1px;
border-color: grey;
background-color: #edeae8;
margin-top:2px;
border-radius : 5px;
}

QLineEdit:hover{
	background-color : #e0e0e0;
}


QLineEdit:focus{
	background-color:#e0e0e0;
	border-width : 3px;
	border-color:#f2ce57;
}</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>High Value 1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="low2">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
border-style:solid;
border-width: 1px;
border-color: grey;
background-color: #edeae8;
margin-top:2px;
border-radius : 5px;
}

QLineEdit:hover{
	background-color : #e0e0e0;
}


QLineEdit:focus{
	background-color:#e0e0e0;
	border-width : 3px;
	border-color:#f2ce57;
}</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Low Value 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="high2">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
border-style:solid;
border-width: 1px;
border-color: grey;
background-color: #edeae8;
margin-top:2px;
border-radius : 5px;
}

QLineEdit:hover{
	background-color : #e0e0e0;
}


QLineEdit:focus{
	background-color:#e0e0e0;
	border-width : 3px;
	border-color:#f2ce57;
}</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>High Value 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: black;
	image : url(:/new/prefix1/remapper.png);
	border: 1px solid white; 
	border-radius: 10px;
} 

QPushButton:hover{
	
	background-color: rgb(102, 102, 102);
	image : url(:/new/prefix1/remapper.png);
	border: 1px solid white; 
	border-radius: 10px;
	color: white;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="output">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Output</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Refresh">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	color:white;
	background-color: black;
	border: 1px solid white; 
	border-radius: 10px;
} 

QPushButton:hover{
	color:white;
	background-color: #ffc400;
	border: 1px solid white; 
	border-radius: 10px;
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>val</tabstop>
  <tabstop>low1</tabstop>
  <tabstop>high1</tabstop>
  <tabstop>low2</tabstop>
  <tabstop>high2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Refresh</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
